docs/topics/readme/cd-cache-thorntail.properties

```diff
@@ -0,0 +1,7 @@
+guideURL=http://launcher.fabric8.io/docs/wf-swarm-runtime.html
+prerequisite=IMPORTANT: This booster requires Java 8 JDK or greater and Maven 3.3.x.
+cicdSection=IMPORTANT: As part of the process of creating this booster, developers.redhat.com/launch or the Fabric8 Launcher set up a project with a CI/CD deployment of this booster. You can see the status of this deployment in your Single-node OpenShift Cluster or OpenShift Online Web Console.
+gitCMD=git clone git@github.com:${loggedUser}/${targetRepository}
+OSORunCMD=mvn clean fabric8:deploy -Popenshift
+app-name=wfswarm-cache-greeting
```

Comment:
Not sure if the app name changes too /cc @Ladicek?

Reply:
Yea, `thorntail-cache-greeting`.